src/main/java/com/intel/gkl/pairhmm/IntelPairHmm.java

```diff
@@ -108,6 +108,9 @@ public void computeLikelihoods(ReadDataHolder[] readDataArray,
                                    HaplotypeDataHolder[] haplotypeDataArray,
                                    double[] likelihoodArray)
```

Comment:
I think "throws NullPointerException " needs to be added to computeLikelihoods?

Reply:
Fixed.